main/res/values/strings.xml

```diff
@@ -703,6 +707,10 @@
     <string name="init_local_file_system">Local File System</string>
     <string name="init_base_directory_description">Base folder for public local data</string>
     <string name="init_map_directory_description">Folder with offline maps</string>
+    <string name="init_brouter_directory_description">Folder with BRouter routing data</string>
+    <string name="init_brouter_directory_additionalinfo">(Default folder is something like \'/brouter/segments4\'. BRouter needs to be configured to use the same folder, see \'storageconfiguration.txt\' in BRouter\'s \'segments4\' folder.)</string>
```

Comment:
> Default folder is something like \'/brouter/segments4\'.

IMHO the most important point is, where the user can find this directory. `/Android/data/btools.routingapp/files/brouter/...`

And what about the `storageconfiguration.txt`? Why is it important or what does it mean? Yes, I found the file in the segments folder but I somehow don't understand it...

Reply:
This is strange: For some reason, BRouter used to store its data in `/brouter/` on some earlier device, and so I transferred this folder from device to device, never recognizing that on newer devices the folder has changed to `/Android/data/btools.routingapp/files/brouter/` :-(
I will change the default string accordingly.

On Android 11+ devices you cannot reach this folder any longer using SAF. You'll need to create a folder in public area to store the download data, and point both c:geo and BRouter to this folder. For that you'll have to edit the `storageconfig.txt` file in `segments4` folder. (But it's probably better to explain this in an FAQ entry instead of in the app.)

Reply:
sounds like quite some effort and only for experts. Do we really want to implement such downloader which will not work unless you tweak external files manually? Or did I miss something...have to admit, that I did not follow the whole thread.

Reply:
The downloader will work easily on systems up to Android 10 - you select the folder (same as you do with map folder etc.), and you are ready to go. 

More restrictive SAF handling in Android 11 makes things a bit more complicated, though. You'll have to tweak a folder setting in BRouter configuration then. Same thing that you had to do when you wanted to store the BRouter files on an SD card.

Reply:
Anyway, as mentioned in the original issue, I'll try to contact the author of BRouter to see if he'll accept PR for BRouter. We could remove the download throttling in BRouter, and we could add a background downloading of tiles then, the same way as we do in c:geo now. My goal for this would be to trigger a download from c:geo in BRouter, and avoid the storage related settings in c:geo by this. But that's some dreams of the future...

Reply:
Might be a misunderstanding - this PR uses SAF to access the routing data files, on both the receiving and the writing end. 

The "Android 11 restrictions" come from a different side: Starting with Android 11 you cannot access app-internal folders for a different app any longer, they are simply not listed any longer in the SAF selector. That's why in that case you need to configure BRouter to use a different folder, which is still reachable by other apps (using SAF). BRouter has a config file for doing exactly such a thing.

Reply:
Indeed, a misunderstanding :) I am relaxed now ;)